src/logger/logger.ts

```diff
@@ -81,8 +75,13 @@ export class IsomerLogger implements ExtendedLogger {
 }
 
 const logger = new IsomerLogger()
-if (useConsoleLogger) logger.use(consoleLogger)
-if (useCloudwatchLogger) logger.use(new CloudWatchLogger())
-logger.useFormatter((message) => `${timestampGenerator()}: ${message}`)
+
+if (!USE_CONSOLE_LOGGER) {
```

Comment:
nit: which is there 2 if statements, can just combine to 1 if else?

Reply:
done